src/scripts/main.js

```diff
@@ -358,3 +358,26 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+const tableBody = document.querySelector('table > tbody');
```

Comment:
Ensure that the `tableBody` element is correctly selected and exists in the DOM. This line assumes that there is a `<tbody>` element inside a `<table>` in the HTML structure.